app/models/card.rb

```diff
@@ -48,17 +48,18 @@ def set_review_interval_wrong_answers
     end
   end
 
-  def number_of_typos(answer)
-    DamerauLevenshtein.distance(original_text, answer, 0)
-  end
-
-
   def self.create_card_in_deck(user, params)
     deck = find_or_create_deck(params[:deck_name], user)
 
     deck.cards.build(params)
   end
 
+  def self.notify_user_for_review_pending_cards
+    Card.where(review_date: Time.now).each do |card|
```

Comment:
Use `find_each` instead of `each`.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.